plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -1695,6 +1695,25 @@ def calculate_checksum(file_handle):
         """
         return file_utils.calculate_checksum(file_handle)
 
+    @property
+    def rpm_search_dicts(self):
+        """
+        Return a list of dictionaries with NEVRA data for each artifact (RPM) this modulemd includes
+
+        :return:            A list of of dicts that contain one NEVRA for each artifact
+        :rtype:             list
+        """
+        rpm_dicts = []
```

Comment:
It makes sense to have a generator here rather than a list because we don't know how many rpms can be in a module.
With errata it's different. It's very unlikely that to solve one specific problem one will need to update a lot of rpms.

I took a look how rpm_search_dicts is used in the code and the assumption that it's a list is quite strong :(